in_toto/cmd/verify.go

```diff
@@ -41,8 +43,29 @@ of link metadata files named ‘<step name>.<functionary keyid prefix>.link’.`
 			layoutKeys[pubKey.KeyID] = pubKey
 		}
 
+		intermediatePems := make([][]byte, 0, len(intermediatePaths))
+		for _, intermediate := range intermediatePaths {
+			f, err := os.Open(intermediate)
+			if err != nil {
+				fmt.Println(fmt.Sprintf("failed to open %v: %v", intermediate, err))
+				os.Exit(1)
+			}
+
+			pemBytes, err := ioutil.ReadAll(f)
+			if err != nil {
+				fmt.Println(fmt.Sprintf("failed to read %v: %v", intermediate, err))
+				os.Exit(1)
+			}
+
+			intermediatePems = append(intermediatePems, pemBytes)
+			if err := f.Close(); err != nil {
+				fmt.Println(fmt.Sprintf("failed to read %v: %v", intermediate, err))
+				os.Exit(1)
+			}
+		}
+
```

Comment:
This whole bit should probably be in it's own function but I think we need to look at how we're loading keys and certs in the bigger picture.